.circleci/config.yml

```diff
@@ -314,9 +311,8 @@ jobs:
 
   cli-smoke-test:
     docker:
-      - image: circleci/node:17
+      - image: cimg/node:current
```

Comment:
Not sure why I had this one set specifically to 17. It should have been just latest version.